en/docs/cli/self-update.md

```diff
@@ -8,4 +8,6 @@ layout: guide
 
 ##### `yarn self-update` <a class="toc" id="toc-yarn-self-update" href="#toc-yarn-self-update"></a>
 
+*Note: `self-update` is not available in version 0.18.1, which means updates must be done manually. See <a href="https://github.com/yarnpkg/yarn/issues/1139">issue #1139</a> for details.*
```

Comment:
Maybe just omit the version number and say "self-update is currently not available"